designer/server/src/main/scala/pl/touk/nussknacker/ui/api/TestingApiErrorMessages.scala

```diff
@@ -0,0 +1,41 @@
+package pl.touk.nussknacker.ui.api
+
+object TestingApiErrorMessages {
+
+  val noSourcesWithTestDataGeneration = "No sources with test data generation available"
+
+  val noDataGenerated = "No data was generated"
+
+  val noInputRecords = "No input records found"
+
+  def requestedTooManySamplesToGenerate(maxSamples: Int) =
+    s"Too many samples requested, limit is $maxSamples. Please configure 'testDataSettings.maxSamplesCount' to increase the limit"
+
+  def tooManyCharactersGenerated(length: Int, limit: Int) =
+    s"$length characters were generated, limit is $limit. Please configure 'testDataSettings.testDataMaxLength' to increase the limit"
+
+  def tooManyCharactersReceived(length: Int, limit: Int) =
+    s"Received $length characters, limit is $limit. Please configure 'testDataSettings.testDataMaxLength' to increase the limit"
```

Comment:
We should talk to user from their perspective, not us. So it should be rather passed instead of received, but personally I would skip this part and focus on where these characters were I mean:
"Test data/sample has too many characters ..."

Reply:
Done